src/beeware_theme/_static/beeware-theme.css

```diff
@@ -157,50 +364,51 @@ div.table {
     column-gap: 2rem;
 }
 
-div.table > * {
-    width: 20rem;
-}
+    div.table > * {
```

Comment:
Is this indentation (and possibly a closing brace at the bottom) intentional? It looks like it might have been part of a media declaration at some point, but isn't in the final version.

Reply:
I actually have no idea what happened here. I thought it might have been from me fixing the Bootstrap code spacing, but that doesn't explain the superfluous closing bracket. Good catch. Fixing.